src/snmalloc/backend_helpers/pagemap.h

```diff
@@ -325,4 +325,81 @@ namespace snmalloc
       body[p >> SHIFT] = t;
     }
   };
+
+  template<
+    typename PAL,
+    typename ConcreteMapT,
+    typename EntryT,
+    bool fixed_range>
+  class Pagemap
+  {
+  public:
+    /**
+     * Export the type stored in the pagemap.
+     */
+    using Entry = EntryT;
+
+    SNMALLOC_REQUIRE_CONSTINIT
+    static inline ConcreteMapT concretePagemap;
```

Comment:
Please add a doc comment and a blank line after the end.

Reply:
This should also be `private`.  Only the back end class should touch it directly (and then only for initialisation).